pkg/coveraggregator/repos.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2024 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package coveraggregator
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/google/syzkaller/pkg/vcs"
+	"github.com/google/syzkaller/sys/targets"
+)
+
+func GetFileVersions(targetFilePath string, rbcs []RepoBranchCommit) map[RepoBranchCommit]string {
```

Comment:
As far as I understand the package public interface is only the AggregateStreamData function, if so this shouldn't be exported.